Training/python/scaling_utils.py

```diff
@@ -173,11 +173,11 @@ def compute_mean(sums, counts, aggregate=True, *file_range):
     if aggregate:
         if file_range:
             assert len(file_range) == 2 and file_range[0] <= file_range[1]
-            return sums[file_range[0]:file_range[1]].sum()/counts[file_range[0]:file_range[1]].sum()
+            return sums[file_range[0]:file_range[1]].sum()/counts[file_range[0]:file_range[1]].sum() if counts[file_range[0]:file_range[1]].sum()>1 else 0.
         else:
-            return sums.sum()/counts.sum()
+            return sums.sum()/counts.sum() if counts.sum()>1 else 0.
     else:
-        return sums/counts
+        return sums/counts if counts>1 else 0.
```

Comment:
As a minor thing, since `counts` is an array, having it in if statement won't return a single bool (but an array of bool), so I think here (and similarly in `compute_std()`) there should be `counts.sum()>1` too (or `any(counts)>1`):
```suggestion
        return sums/counts if counts.sum()>1 else 0.
```